### DISCUSSION TITLE
Had a problem upgrading to the paid one

### DISCUSSION THREAD

**Discussion by hientt53** [MobiusSync/MobiusSync#124] [General]
Hey, I got two things:
1. I used the free app, then bought in it. Does that mean I got everything the paid app offers?
2. Even after paying in free app, when I pick the paid MobiusSync in the app store, they're asking for money again. Why's that? Thought I'd get it for free now.

<img width="435" alt="image" src="https://github.com/MobiusSync/MobiusSync/assets/7857371/f5f16480-371e-40b6-8b3a-26e74691d653">
<img width="402" alt="image" src="https://github.com/MobiusSync/MobiusSync/assets/7857371/6721bc6e-efec-4416-8d9c-0f0853637824">

**Comment by kwlorenz** [comment]
Because the open source project "Syncthing" (which is the backbone of Mobius Sync) is run by volunteers. So you can donate to the "Syncthing" project and not to Mobius Sync ...

**Comment by egigoka** [comment]
If you purchased in app, you're alright, no need to download Pro version. It's for businesses that can't purchase in-app.
